### PR TITLE
Adding units to the Metrics API, adding one attribute to the DiagnosticSettings API, updating documentation

### PR DESCRIPTION
**Pull request by gucalder**
- Adding some metric unit names to the **Metrics API 2018-01-01**. The complete list now includes the following: "Count", "Bytes", "Seconds", "CountPerSecond", "BytesPerSecond", "Percent", "MilliSeconds", "ByteSeconds", "Unspecified", "Cores", "MilliCores", "NanoCores", "BitsPerSecond"

- Adding support for a new attribute in the DiagnosticSettings object (**Diagnostic Settings API 2017-05-01-preview**,) i.e.: the attribute is "logAnalyticsDestinationType," which is a string that accepts the following values: Dedicated or null

- Minor modification to the documentation

### CODE REVIEW

File: src/SDKs/_metadata/monitor_resource-manager.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+2019-08-01 03:08:36 UTC
```
Comment (erich-wang) [inline src/SDKs/_metadata/monitor_resource-manager.txt]: @gucalder, this file should be under **eng\mgmt\mgmtmetadata**

**Review by erich-wang — CHANGES_REQUESTED** [review]

File: src/SDKs/_metadata/monitor_resource-manager.txt (hunk 1 above)
Comment (gucalder) [reply src/SDKs/_metadata/monitor_resource-manager.txt]: Ok, moving the file.
BTW: this file was created automatically. There has to be some kind of bug or I am running the wrong command to generate the code.

**Comment by gucalder** [comment]
Ping

File: src/SDKs/_metadata/monitor_resource-manager.txt (hunk 1 above)
Comment (erich-wang) [reply src/SDKs/_metadata/monitor_resource-manager.txt]: @gucalder , you need to first pull latest code from master, then follow the step 2 https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#sdk-generation-from-updated-spec
**msbuild eng/mgmt.proj /t:Util /p:UtilityName=InstallPsModules**

**Review by erich-wang — APPROVED** [review]

**Comment by erich-wang** [comment]
@gucalder , overall LGTM, could you provide the swagger spec PR link related to this PR? I need to confirm whether the spec is merged?

**Comment by gucalder** [comment]
These are the links to the PRs for the last specs changes:
https://github.com/Azure/azure-rest-api-specs/pull/6485
https://github.com/Azure/azure-rest-api-specs/pull/6454

File: sdk/monitor/Microsoft.Azure.Management.Monitor/changelog.md [hunk 2]
```diff
@@ -1,5 +1,25 @@
...
+
+- Adding some metric unit names to the **Metrics API 2018-01-01**. The complete list now includes the following: "Count", "Bytes", "Seconds", "CountPerSecond", "BytesPerSecond", "Percent", "MilliSeconds", "ByteSeconds", "Unspecified", "Cores", "MilliCores", "NanoCores", "BitsPerSecond"
+- Adding support for a new attribute in the DiagnosticSettings object (**Diagnostic Settings API 2017-05-01-preview**,) i.e.: the attribute is "logAnalyticsDestinationType," which is a string that accepts the following values: Dedicated or null
+- Minor modification to the documentation
+
+### Changes in 0.24.0-preview
+
+**Notes**
+
+- Added Dynamic Treshold for Metric Alert API and its Unit tests and Scenario Tests.
```
Comment (loic-sharma) [inline sdk/monitor/Microsoft.Azure.Management.Monitor/changelog.md]:
```suggestion
- Added Dynamic Threshold for Metric Alert API and its Unit tests and Scenario Tests.
```

Comment (gucalder) [reply sdk/monitor/Microsoft.Azure.Management.Monitor/changelog.md]: Fixed.

**Comment by gucalder** [comment]
Ping

**Pull request merged**